examples/java/tech/fastj/examples/bullethell/scripts/BulletMovement.java

```diff
@@ -50,7 +50,7 @@ public void destroy() {
 
     public void bulletDied(GameObject obj) {
         FastJEngine.runAfterUpdate(() -> {
-            FastJEngine.log("death! of bullet " + travelAngle + "f " + travelVector);
+            FastJEngine.log("death! of bullet {} f {}", travelAngle, travelVector);
```

Comment:
As mentuoned above, there's an extra space between the first brackets and the `f`.